python/common/org/python/types/Map.java

```diff
@@ -0,0 +1,35 @@
+package org.python.types;
+
+public class Map extends org.python.types.Object implements org.python.Iterable {
+    private org.python.Object callable;
+    private java.util.List<org.python.Iterable> iterators;
```

Comment:
Since the size of this does not change, would it be better to create an array (`[]`) instead of an `ArrayList`?